packages/storybook/package.json

```diff
@@ -25,39 +25,27 @@
     "@ni/nimble-tokens": "*",
     "@ni/spright-components": "*",
     "@ni/spright-react": "*",
-    "@rollup/plugin-node-resolve": "^16.0.0",
-    "@storybook/addon-a11y": "^8.4.3",
-    "@storybook/addon-actions": "^8.4.3",
-    "@storybook/addon-docs": "^8.4.3",
-    "@storybook/addon-essentials": "^8.4.3",
-    "@storybook/addon-interactions": "^8.4.3",
-    "@storybook/addon-links": "^8.4.3",
-    "@storybook/addon-webpack5-compiler-swc": "^3.0.0",
-    "@storybook/cli": "^8.4.3",
-    "@storybook/components": "^8.6.12",
+    "@storybook/addon-a11y": "9.0.5",
+    "@storybook/addon-docs": "9.0.5",
+    "@storybook/addon-links": "9.0.5",
+    "@storybook/cli": "9.0.5",
     "@storybook/csf": "^0.1.11",
-    "@storybook/html": "^8.4.3",
-    "@storybook/html-webpack5": "^8.4.3",
-    "@storybook/manager-api": "^8.4.3",
-    "@storybook/theming": "^8.4.3",
+    "@storybook/html-vite": "9.0.5",
+    "@storybook/builder-vite": "9.0.5",
     "apache-arrow": "20.0.0",
-    "circular-dependency-plugin": "^5.2.0",
     "js-beautify": "^1.15.1",
     "prettier-eslint": "^16.3.0",
     "prettier-eslint-cli": "^8.0.1",
     "react-syntax-highlighter": "^15.6.1",
     "remark-gfm": "^4.0.0",
     "rollup": "^4.12.0",
-    "storybook": "^8.4.3",
-    "storybook-addon-pseudo-states": "^4.0.2",
-    "terser-webpack-plugin": "^5.3.10",
-    "ts-loader": "^9.2.5",
-    "tsconfig-paths-webpack-plugin": "^4.2.0",
-    "typescript": "~5.4.5"
+    "storybook": "9.0.5",
+    "storybook-addon-pseudo-states": "^9.0.5",
+    "typescript": "~5.4.5",
+    "vite":"6.3.5"
```

Comment:
should update to vite 7 which looks supported: https://github.com/storybookjs/storybook/blob/38bd82485c366e5f12dc803583ec3d4b47700392/code/builders/builder-vite/package.json#L63